src/docs/patterns/appbar.mdx

```diff
@@ -1,45 +1,133 @@
+    />
+    <Divider />
+</React.Fragment>
+
+# Collapsible App Bar
 
 <ImageGrid
-    images={[Image1, Image2]}
-    caption={'A collapsible app bar shrinks and gets pinned to the top when the page is scrolled.'}
+    images={[Collapsible, Collapsed]}
+    gridImageProps={{ style: { maxHeight: 265, overflow: 'hidden' } }}
+    caption={['The middle row of text shrinks in size and opacity until it disappears.']}
 />
-<DemoCard repository={'collapsible-appbar'} angular={'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Fapp-bar%2Fcollapsible-app-bar%2Fcollapsible-app-bar.component.ts'} react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/app-bar/collapsible-app-bar/index.tsx&initialpath=collapsible'} ionic reactNative />
 
-In some situations, you may want to present more information in an app bar, such as a large graphic, buttons, or multiple lines of text. In these cases, you can use a taller app bar that will shrink as the user scrolls down the page and be replaced by a basic app bar. This allows you the flexibility to present an eye-catching banner without permanently reducing the amount of usable space on the page. This is a pattern most commonly used for mobile applications, but could also be used at a larger scale.
+In some situations, you may want to present more than just a title and subtitle in an app bar, such as a large graphic, buttons, or multiple lines of text. In these cases, you can use a collapsible app bar that starts out tall but will shrink as the user scrolls down the page until it becomes a basic app bar. This allows you the flexibility to present an eye-catching banner without permanently reducing the amount of usable space on the page. This is a pattern most commonly used for mobile applications, but could also be used for web.
 
-## Search Bar
+The collapsible appbar should transition smoothly between the expanded and collapsed states — this means you will need to account for how the app bar will look throughout the transition. Because elements will disappear as the app bar shrinks, you'll need to use various transition techniques to do this smoothly, depending on your content.
+
+<React.Fragment>
+    <DemoCard
+        repository={'collapsible-appbar'}
+        angular={
+            'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Fapp-bar%2Fcollapsible-app-bar%2Fcollapsible-app-bar.component.ts'
+        }
+        react={
+            'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/app-bar/collapsible-app-bar/index.tsx&initialpath=collapsible'
+        }
+        ionic
+        reactNative
+    />
+    <Divider />
+</React.Fragment>
+
+# Search Bar
 
 <ImageGrid
-    images={[Image3, Image4]}
+    regularWidth
+    images={[Search]}
     caption={'When the search icon is clicked, a white search bar will slide in and cover the default tool bar. '}
```

Comment:
We've got two actionable icons here, both indicate some dismissal actions. Here's what I suggest for the animation:
![search-bar](https://user-images.githubusercontent.com/8997218/92952989-e5d78080-f42e-11ea-84b3-7e5becf727b1.gif)

Reply:
Source file in Figma, if you want to make changes or remove that bottom stroke there